cmd/livepeer_bench/livepeer_bench.go

```diff
@@ -39,6 +39,7 @@ func main() {
 	detectionFreq := flag.Int("detectionFreq", 0, "Run content-detection on every nth segment. No detection occurs for default frequency of 0.")
 	detectionSampleRate := flag.Uint("detectionSampleRate", 1, "Run content-detection on every nth frame of a particular segment, if detectionFreq > 0.")
 	concurrentSessionDelay := flag.Duration("concurrentSessionDelay", 300*time.Millisecond, "Delay before starting a new concurrent session")
+	sign := flag.Bool("sign", false, "Calculate binary signature while transcoding")
```

Comment:
What do you think about using a more descriptive name for the flag in order to avoid any confusion with cryptographic signing (which the node already does automatically) of transcoded results?

```suggestion
	sign := flag.Bool("mpeg7Sign", false, "Calculate MPEG-7 video signature while transcoding")
```